app/src/main/java/waiter/RouteBuilder.java

```diff
@@ -20,37 +17,17 @@ public RouteBuilder url(String url) {
         return this;
     }
 
-    public RouteBuilder method(String method) {
-        this.method = method;
+    public RouteBuilder methods(String[] methods) {
+        this.methods = methods;
         return this;
     }
 
-    public RouteBuilder status(String status) {
-        this.status = status;
+    public RouteBuilder handler(Callable<Response> handler) {
```

Comment:
Do we expect route without a method and/or handler?

Reply:
Im not sure I understand, apologies 😦, what do you mean? Thank you for all the help as well btw! 😄

Reply:
For example, can I make a route without a handler:
```java
RouteBuilder
  .newUp()
  .methods(...)
  .build()
```

Reply:
That is a good point, I can and i don't think I should be able to, I feel like url, methods, and handler should all be required, Im gonna look into making them required (if that makes sense), incredible catch, thank you a ton Steven! 😄

Reply:
Yea I think maybe a route builder might not make sense in this case. Because We always expect `url`, `method` and `handler` to be there so maybe a normal function/method would do:
```java
routes.addRoute(url: ..., method: ..., handler: ...) {}
```

Reply:
Remove routeBuilder addressed in [this commit](https://github.com/jonmanahan/waiter/pull/18/commits/048ea1c654dd2bf3e04dc7d587e08ab8c89648be)